lib/danger/danger_core/environment_manager.rb

```diff
@@ -82,18 +82,18 @@ def raise_error_for_no_request_source(env, ui)
 
     private
 
-    def get_repo_source(repo_url)
-      if repo_url =~ /github/i
+    def get_repo_source()
+      if defined? DANGER_GITHUB_API_BASE_URL
```

Comment:
This is optional I'm afraid, maybe these could be a second check inside the `else` at the end?